src/client/caching/structures/channels/DMChannel.ts

```diff
@@ -50,6 +53,27 @@ export class DMChannel extends Channel {
 		this.messages = new MessageStore(client, this);
 	}
 
+	/**
+	 * Awaits a group of messages
+	 * @param limit The limit of filtered messages to await
+	 * @param options The options to control what you receive
+	 */
+	public async awaitMessages(limit: number, options: MessageCollectorOptions): Promise<Cache<string, Message>> {
+		return new MessageCollector(this, limit, options).collectAll();
+	}
+
+	/**
+	 * Closes the channel.
+	 * @since 0.0.1
+	 * @param requestOptions The additional request options.
+	 * @see https://discord.com/developers/docs/resources/channel#deleteclose-channel
+	 */
+	public async remove(requestOptions: RequestOptions = {}): Promise<this> {
```

Comment:
All deletable structures have their method named `delete`, this should be no exception, and sorry I know it's my code, I forgot ❤.
```suggestion
	public async delete(requestOptions: RequestOptions = {}): Promise<this> {
```